src/_pytest/parser.py

```diff
@@ -0,0 +1,107 @@
+"""
+Find a test class and/or test method in given source code line.
+
+Original code by Pavel Savchenko
+"""
+import ast
+
+class PytestParser(ast.NodeVisitor):
```

Comment:
nitpick, but `PytestParser` doesn't tell much about what the class does, perhaps `TestLocator`?

I know you mentioned that we need to add docs already, just to highlight that we should have unittests for this class as well, in addition to 1 or more integration tests of the feature. 👍